spyderlib/plugins/shortcuts.py

```diff
@@ -807,6 +816,22 @@ def apply_settings(self, options):
         self.table.save_shortcuts()
         self.main.apply_shortcuts()
 
+    def load_button_clicked(self, change):
+        selected = self.combo_keybindings.currentText().lower()
+        reset_shortcuts()

```

Comment:
Ok, let's leave this for another PR, as @blink1073 mentioned.

However, I don't know if a `custom` option is enough. What if you want the Emacs shortcuts but do some customizations on top of them? And also for the Default ones?